### PR TITLE
[wip] Make ntpd listen on admin interface only

### PR DESCRIPTION
**Pull request by bkutil**

### CODE REVIEW

**Comment by bkutil** [comment]
cc @vuntz

**Comment by vuntz** [comment]
Not tested, but looks good to me.

**Comment by bkutil** [comment]
Good to know this is not way off, thanks :) I'll give it a test.

**Comment by jsuchome** [comment]
Shouldn't it beSUSE only ?

**Comment by bkutil** [comment]
@jsuchome fixed

**Comment by jsuchome** [comment]
I like it (I'm not ntp expert, but it looks correct)

**Comment by bkutil** [comment]
Looks, like -I does not have any effect at all. The 'interface listen'/'interface ignore' works, but only on SLE12.

**Comment by tboerger** [comment]
LGTM

File: chef/cookbooks/ntp/templates/default/ntp.conf.erb [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 driftfile <%= @driftfile %>
 
+interface ignore wildcard
+interface listen <%= @admin_interface %>
```
Comment (vuntz) [inline chef/cookbooks/ntp/templates/default/ntp.conf.erb]: I just checked, this got added in ntp 4.2.5p212. SLES11 has 4.2.4p8.

**Comment by vuntz** [comment]
+1

File: chef/cookbooks/ntp/templates/default/ntp.conf.erb (hunk 1 above)
Comment (bkutil) [inline chef/cookbooks/ntp/templates/default/ntp.conf.erb]: Yep, noticed that too. Should I try to do a backport?

Comment (vuntz) [inline chef/cookbooks/ntp/templates/default/ntp.conf.erb]: No, not worth it. Was just documenting here (for myself) why it won't work on SLES11 :-)

**Pull request merged**